bin/alias.bat

```diff
@@ -1,8 +1,8 @@
 @echo off
 
 
-if "%aliases%" == "" (
-  set ALIASES=%CMDER_ROOT%\config\user-aliases.cmd
+if "%ALIASES%" == "" (
+  set aliases="%CMDER_ROOT%\config\user-aliases.cmd"
```

Comment:
Is this supposed to be lowercase?

Reply:
Yeah that looks a bit weird

Reply:
I can change it to either be upper case or lower case - Command prompt variables are case insensitive.

The original problem was that half the file was using an upper case variant of the name, while the rest was using a lower case variant.

Reply:
well imo it should be consistent if you change it now, change it everywhere, no?

Reply:
Can you change it?